website/blog/2024-02-13-run-mongodb-workloads-ferretdb-tembo.md

```diff
@@ -0,0 +1,227 @@
+
+#### Example 2: Finding Data for a Specific Period
+
+To analyze our supply data over specific periods, we can query the collection record within the desired date ranges.
+This could be useful in understanding the impact of a particular promotional campaign.
+
+```js
+db.supply.find({
+  transaction_time: {
+    $gte: ISODate('2023-07-01T00:00:00.000Z'),
+    $lte: ISODate('2023-07-03T23:59:59.000Z')
+  }
+})
+```
+
+Result:
+
+```json5
+[
+  {
+    _id: ObjectId('65c0d6d5cfba3092d3958fdb'),
+    transaction_id: 10032,
+    customer_name: 'Emma Olsen',
+    customer_location: 'Bergen, Norway',
+    product_category: 'Electronics',
+    transaction_time: ISODate('2023-07-01T14:00:00.000Z'),
+    product_name: 'Amazon Echo',
+    price: 99,
+    quantity: 1,
+    payment_method: 'Bank Transfer'
+  },
+  {
+    _id: ObjectId('65c0d6d5cfba3092d3958fdc'),
+    transaction_id: 10033,
+    customer_name: 'Jens Petersen',
+    customer_location: 'Copenhagen, Denmark',
+    product_category: 'Fashion',
+    transaction_time: ISODate('2023-07-02T15:00:00.000Z'),
+    product_name: 'Fossil Watch',
+    price: 150,
+    quantity: 1,
+    payment_method: 'MobilePay'
+  },
+  {
+    _id: ObjectId('65c0d6d5cfba3092d3958fea'),
+    transaction_id: 10047,
+    customer_name: 'Sarah Johnson',
+    customer_location: 'Sydney, Australia',
+    product_category: 'Electronics',
+    transaction_time: ISODate('2023-07-03T13:00:00.000Z'),
+    product_name: 'Dell Laptop',
+    price: 1200,
+    quantity: 1,
+    payment_method: 'Credit Card'
+  },
+  {
+    _id: ObjectId('65c0d6d5cfba3092d3958feb'),
+    transaction_id: 10048,
+    customer_name: 'Mehmet Yilmaz',
+    customer_location: 'Istanbul, Turkey',
+    product_category: 'Fashion',
+    transaction_time: ISODate('2023-07-03T14:00:00.000Z'),
+    product_name: 'Handmade Bag',
+    price: 80,
+    quantity: 1,
+    payment_method: 'Debit Card'
+  }
+]
+```
+
+This query filters the records to only include transactions that occurred between the 1st and 3rd of July 2023.
+
+#### Example 3: Identifying the most bought Item
+
+Say you want insight into the most in-demand item.
+We can aggregate the data to sum the quantities sold by product name.
+
+```js
+db.supply.aggregate([
+  {
+    $group: {
+      _id: '$product_name',
+      totalQuantity: { $sum: '$quantity' }
+    }
+  },
+  { $sort: { totalQuantity: -1 } },
+  { $limit: 1 }
+])
+```
+
+Result:
+
+```json5
+[{ _id: 'The Alchemist', totalQuantity: 5 }]
+```
+
+This operation groups the records by `product_name`, sums up the quantities, and then sorts the results in descending order of quantity to highlight the most popular item.
+
+These are just a few examples of how FerretDB can run MongoDB workloads for many use cases.
+You can go ahead to try other operations on FerretDB.
+
+## Get started with FerretDB on Tembo
+
+Running FerretDB on Tembo gives you access to high-performance, fully extensible managed Postgres service.
```

Comment:
Would be better: 

Running FerretDB enables you to run MongoDB workloads on a high-performance, fully extensible managed Postgres service.